gfx/gfx_animation.c

```diff
@@ -1848,15 +1849,17 @@ bool gfx_animation_line_ticker(gfx_animation_ctx_line_ticker_t *line_ticker)
       goto end;
 
    /* Line wrap input string */
-   wrapped_str = (char*)malloc((strlen(line_ticker->str) + 1) * sizeof(char));
+   wrapped_str_len = strlen(line_ticker->str) + 1 + 10; /* 10 bytes use for inserting '\n' */
+   wrapped_str = (char*)malloc(wrapped_str_len);
    if (!wrapped_str)
       goto end;
 
```

Comment:
Could you please NUL terminate `wrapped_str` here? (i.e. `wrapped_str[0] = '\0';`)

In the RetroArch codebase, it is a coding standard that we always NUL terminate strings before first use